src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,81 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        final DriverService driverService
+                = (DriverService) injector.getInstance(DriverService.class);
+        final ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+        // TESTING CREATE METHOD
+        Driver firstDriver = new Driver();
+        firstDriver.setName("Alex");
+        firstDriver.setLicenseNumber("1224");
+        Driver driverAlex = driverService.create(firstDriver);
+        Manufacturer firstManufacturer = new Manufacturer();
+        firstManufacturer.setCountry("Ukraine");
+        firstManufacturer.setName("first_name");
+        Manufacturer manufacturerFromUkraine = manufacturerService.create(firstManufacturer);
+
+        // TESTING CREATE METHOD
+        Driver secondDriver = new Driver();
+        secondDriver.setName("Alice");
+        secondDriver.setLicenseNumber("1240");
+        Driver driverAlice = driverService.create(secondDriver);
+        Manufacturer secondManufacturer = new Manufacturer();
+        secondManufacturer.setCountry("Spain");
+        secondManufacturer.setName("second_name");
+        Manufacturer manufacturerFromSpain = manufacturerService.create(secondManufacturer);
+
+        // TESTING CREATE METHOD
+        Driver thirdDriver = new Driver();
+        thirdDriver.setName("Bob");
+        thirdDriver.setLicenseNumber("2902");
+        Driver driverBob = driverService.create(thirdDriver);
+        Manufacturer thirdManufacturer = new Manufacturer();
+        thirdManufacturer.setCountry("Lithuania");
+        thirdManufacturer.setName("third_name");
+        Manufacturer manufacturerFromLithuania = manufacturerService.create(thirdManufacturer);
```

Comment:
```suggestion
        // TESTING DRIVER CREATE METHOD
        Driver firstDriver = new Driver();
        firstDriver.setName("Alex");
        firstDriver.setLicenseNumber("1224");
        Driver driverAlex = driverService.create(firstDriver);
        
        Driver secondDriver = new Driver();
        secondDriver.setName("Alice");
        secondDriver.setLicenseNumber("1240");
        Driver driverAlice = driverService.create(secondDriver);

        Driver thirdDriver = new Driver();
        thirdDriver.setName("Bob");
        thirdDriver.setLicenseNumber("2902");
        Driver driverBob = driverService.create(thirdDriver);
        
        // TESTING MANUFACTURER CREATE METHOD
        Manufacturer firstManufacturer = new Manufacturer();
        firstManufacturer.setCountry("Ukraine");
        firstManufacturer.setName("first_name");
        Manufacturer manufacturerFromUkraine = manufacturerService.create(firstManufacturer);

        Manufacturer secondManufacturer = new Manufacturer();
        secondManufacturer.setCountry("Spain");
        secondManufacturer.setName("second_name");
        Manufacturer manufacturerFromSpain = manufacturerService.create(secondManufacturer);
       
        Manufacturer thirdManufacturer = new Manufacturer();
        thirdManufacturer.setCountry("Lithuania");
        thirdManufacturer.setName("third_name");
        Manufacturer manufacturerFromLithuania = manufacturerService.create(thirdManufacturer);
```
so it reads a little better, agree) I suggest doing the same with the following cases (Optional)